### ISSUE TITLE
[Question] dipy/denoise/gibbs.py

### ISSUE THREAD

**Issue by tashrifbillah** [dipy/dipy#2754]
https://github.com/dipy/dipy/blob/77afd6098c70939dde1647e71fa1d6c7be1ef48e/dipy/denoise/gibbs.py#L322-L324

The copied volume is overwritten below anyway:

https://github.com/dipy/dipy/blob/77afd6098c70939dde1647e71fa1d6c7be1ef48e/dipy/denoise/gibbs.py#L327-L335

So what is the point of using `inplace` parameter?

**Comment by skoudoro** [comment]
Good catch, I think this parameter is useless. Either we need to remove the parameter or make ` inplace=False` work.  Low priority. 

Feel free to fix this issue @tashrifbillah !

**Comment by skoudoro** [comment]
After playing with the code, I think I made a mistake. The `inplace` argument works as expected and avoid you to modify the input volume. 

So closing the issue and the PR
